modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -200,19 +207,21 @@ function getRedeemIndex(keyCombinations: [Buffer, Buffer][], signer: Buffer, cos
   throw new Error(`could not find singer/cosigner combination`);
 }
 
-export function createPaymentP2tr(
+function createPaymentP2trCommon(
   pubkeys: Triple<Buffer>,
-  redeemIndex?: number | { signer: Buffer; cosigner: Buffer }
+  redeemIndex?: number | { signer: Buffer; cosigner: Buffer },
+  isMusig2 = false
 ): bitcoinjs.Payment {
   const keyCombinations2of2 = getTaptreeKeyCombinations(pubkeys);
+  const scriptPathKeyCombinations2of2 = isMusig2 ? keyCombinations2of2.slice(1) : keyCombinations2of2;
   if (typeof redeemIndex === 'object') {
-    redeemIndex = getRedeemIndex(keyCombinations2of2, redeemIndex.signer, redeemIndex.cosigner);
+    redeemIndex = getRedeemIndex(scriptPathKeyCombinations2of2, redeemIndex.signer, redeemIndex.cosigner);
```

Comment:
github shows this in a strange way, looks normal locally
![image](https://user-images.githubusercontent.com/283533/219390687-aeaff48c-608d-4e26-a6f6-0e36830ebdaa.png)